pythran/tests/test_numpy_fft.py

```diff
@@ -40,6 +40,18 @@ def test_rfft_10(self):
     def test_rfft_11(self):
         self.run_test("def test_rfft_11(x,n): from numpy.fft import rfft ; return rfft(x,n)", numpy.arange(0,8.).astype(numpy.float32),16,test_rfft_11=[NDArray[numpy.float32,:],int])
 
+    # Test parallel:
+    def test_rfft_12(self):
+        self.run_test('''
+import numpy as np
+def test_rfft_12(x):
+    out = []
+    #omp parallel for
+    for ii in range(20):
+        out.append(np.fft.rfft(x))
```

Comment:
unfortunately, this is not thread safe because the list container is not thread safe.
You'd better initialize `out` as `out = [np.empty_like(x, dtype=complex) for i in range(20)]` and then access each element of `out`

Reply:
Ooops, missed that! Should be corrected now.